.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+sudo: false
+python:
+    - 2.7
+install:
+    - pip install -U pip setuptools
+    - pip install -U -e .[test]
+script:
+    - python -m zope.testrunner --test-path=src {posargs:-vc}
```

Comment:
No reason to use `python -m zope.testrunner` instead of `zope-testrunner`.

You may want to consider `coverage run -m zope.testrunner`, and then we can enable coveralls.io integration as well.